### PR TITLE
Rename SendImage.imageUrl -> SendImage.imageUri

### PR DESCRIPTION
**Pull request by blazern**
Renaming SendImage.imageUrl -> SendImage.imageUri for #144

### CODE REVIEW

File: test/api_getProductRaw_test.dart [hunk 1]
```diff
@@ -80,7 +80,7 @@ void main() {
           result.product!.nutrientLevels!.levels[NutrientLevels.NUTRIENT_SALT],
           Level.MODERATE);
       expect(result.product!.countries,
-          'Belgique,France,Allemagne,Pays-Bas,Espagne,Suisse');
+          'Belgique,France,Allemagne,Pays-Bas,Espagne,Suisse, en:world');
```
Comment (blazern) [inline test/api_getProductRaw_test.dart:83]: Doesn't look like it's related to my changes, but the test failed without `en:world`.

**Review by peterwvj — APPROVED** [review]
Looks good, thanks.

File: test/api_getProductRaw_test.dart (hunk 1 above)
Comment (stephanegigandet) [reply test/api_getProductRaw_test.dart:83]: It's because an unrelated change happened on the dev database. I think we should remove that test.

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**